now/executor/gateway/hubble_report.py

```diff
@@ -99,6 +100,7 @@ def report(quantity_basic, quantity_pro, auth_jwt=None):
                     'appId': 'search-app',
                     'feeType': 'search-fee',
                     'source': 'gateway',
+                    'flowId': namespace.split('-')[1] if namespace else namespace,
```

Comment:
you can rely on `user_input.flow_name`
```suggestion
                    'flowId': user_input.flow_name + '-' + namespace,
```